guides/trello-workflow.md

```diff
@@ -0,0 +1,53 @@
+# Trello Workflow
+
+New workflow. Uses Trello.
+
+## States
+
+- `backlog`
+- `in progress`
```

Comment:
Some boards use another list for Tickets. This might be useful when there is a sprint running and we want to see if there is any outstanding work outside of sprint. However, we can also leave these States flexible for each project because each is a bit different.